console/network/src/lib.rs

```diff
@@ -195,6 +195,14 @@ pub trait Network:
     /// The maximum number of certificates in a batch.
     const MAX_CERTIFICATES: u16;
 
+    /// The maximum number of stacks in the process.
+    /// Must be at least Self::MAX_PROGRAM_DEPTH * Self::MAX_IMPORTS to be safe.
+    #[cfg(not(any(test, feature = "test")))]
+    const MAX_STACKS: usize = Self::MAX_PROGRAM_DEPTH * Self::MAX_IMPORTS * 10;
+    /// The maximum number of stacks in the process.
+    #[cfg(any(test, feature = "test"))]
+    const MAX_STACKS: usize = 65;
```

Comment:
is this just a randomly chosen lower number that worked fine, or a minimum below which the tests don't pass?

Reply:
A minimum below which the tests don't pass. But ideally, I should just get rid of this testing constant because future tests might require more (up to `Self::MAX_PROGRAM_DEPTH * Self::MAX_IMPORTS`).